internal/core.go

```diff
@@ -2,15 +2,18 @@ package internal
 
 import (
 	"context"
+	_ "embed"
 	"runtime"
 )
 
 const (
-	SDKSemverVersion      = "0010009" // v0.1.0-beta.9
 	SDKLanguage           = "Go"
 	DefaultRequestLibrary = "net/http"
 )
 
+//go:embed version-build.txt
```

Comment:
nit: I don't think the `.txt` extension is necessary, here. I don't see any immediate disadvantages to it, but naming the file simply `version-build` may be cleaner.

Reply:
Done.

Reply:
I don't think this is done

Reply:
Now its done..